docs/src/how-to/4-develop.md

```diff
@@ -0,0 +1,145 @@
+# Advanced Users
+
+If you really want to stay in tune with the most recent features, test some ideas, or even contribute to the KomaMRI project, it is necessary to install and configure some tools on your machine:
+- **Juliaup:** This is the manager for installing different versions of Julia.
+- **Julia:** This is the programming language. It is advisable to install it with Juliaup.
+- **Git:** This is a version control system handy for coding.
+- **GitHub:** This is a cloud-based Git repository handy for managing the KomaMRI project. You need to create an account.
+- **VScode:** This is a code editor with support for development operations.
+- **VScode Plugins:** They enable VSCode to have more handy features. We recommend you install the following: "Julia", "Git Graph", "GitHub Pull Requests"."
+
+## Setup your Fork of KomaMRI
+
+Once everything is installed, you need to create a fork of the KomaMRI repository in your GitHub account. Go to the official [KomaMRI repository](https://github.com/JuliaHealth/KomaMRI.jl) and follow the steps below:"
+
+![](../assets/dev-fork.png)
+![](../assets/dev-fork-create.png)
+
+Now, your fork of KomaMRI should look similar to the image below:
+
+![](../assets/dev-fork-github.png)
+
+## Setup your Local Git Project
+
+Now, you need to clone your fork of the KomaMRI repository. In Julia, you can do this by opening a Julia REPL, open the package manager with the `]` button and typing the following command (make sure to replace `<github-username>` with your GitHub username):"
+```julia-repl
+(@v1.10) pkg> dev https://github.com/<github-username>/KomaMRI.jl
+```
+
+This saves the local Git project at the `~/.julia/dev/KomaMRI/` directory. You can then open VSCode and navigate to this folder; it should look like this:
```

Comment:
I actually encountered an error when I ran this:
```julia-repl
(@v1.10) pkg> dev https://github.com/JanWP/KomaMRI.jl
     Cloning git-repo `https://github.com/JanWP/KomaMRI.jl`
   Resolving package versions...
ERROR: Unsatisfiable requirements detected for package KomaMRICore [4baa4f4d]:
 KomaMRICore [4baa4f4d] log:
 ├─possible versions are: 0.7.4-0.8.3 or uninstalled
 └─restricted to versions 0.9 by KomaMRI [6a340f8b] — no versions left
   └─KomaMRI [6a340f8b] log:
     ├─possible versions are: 0.9.0 or uninstalled
     └─KomaMRI [6a340f8b] is fixed to version 0.9.0-DEV
```

However, I was able to follow the rest of the instructions just fine, since all that is needed at this stage is to get the source code checked out in to the Julia dev directory. Maybe the same could be achieved without error using a `git checkout`, or a comment could be added to let the reader know that this error can be safely ignored.

Secondly, should the text remind inexperienced users to check they have set up their anonymous GitHub email address for the local repository? Otherwise push may fail later if they opted to keep their email address private in their GitHub settings.

````suggestion
```julia-repl
(@v1.10) pkg> dev https://github.com/<github-username>/KomaMRI.jl
```

This saves the local Git project at the `~/.julia/dev/KomaMRI/` directory. You may get an error regarding unsatisfiable requirements, due to the naming of the development verion of KomaMRI packages (e.g. 0.9.0-DEV). You can ignore the error for now, instructions on how to correctly set up the Julia environment are below.

If you have configured your GitHub account to keep your email address private, make sure, you use your private GitHub email (`<github-username>@users.noreply.github.com`) in your local Git repository. Otherwise GitHub will not allow you to push your local commits to GitHub. To check the current setting for your email:

```bash
user@machine:~/.julia/dev/KomaMRI$ git config user.email
```

You can modify the setting in the following way (make sure to replace `<github-username>` with your GitHub username):

```bash
user@machine:~/.julia/dev/KomaMRI$ git config user.email <github-username>@users.noreply.github.com
```

You can then open VSCode and navigate to this folder; it should look like this:
````